### PR TITLE
Podcast

### PR DESCRIPTION
**Pull request by Florie25**
Hi Anne, this should one should. be the last one for a while.

Thanks.
Florie

### CODE REVIEW

File: _posts/2021-01-02-Build-the-future-podcast.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+thumbnail: images/pulse-thumbnail-build-the-future.jpg
+---
+
+Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from Build The Future, about the [future of underwater wireless communications](https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT).
+
+Build the Future shares the compelling visions of the future and the stories of those building it.
+
+In this episode, Manu also talks about the work we do at Subnero, the state of underwater networks today, marine research, underwater robots, and the future of our oceans.
+
+Plug your earphones in, switch your screen off for half and hour and listen!
```
Comment (AnneThKu) [inline _posts/2021-01-02-Build-the-future-podcast.md]:
```suggestion
Plug your earphones in, switch your screen off for half an hour and listen!
```

**Review by AnneThKu — CHANGES_REQUESTED** [review]

**Comment by AnneThKu** [comment]
Thumbnail issue same as mentioned for SIWW_online PR.

File: _posts/2021-01-02-Build-the-future-podcast.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Podcast with Build The Future
+banner : images/banner-pulse-standard.jpg
+date:  2021-01-02
+categories: wnc news
+thumbnail: images/pulse-thumbnail-build-the-future.jpg
+---
+
+Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from Build The Future, about the [future of underwater wireless communications](https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT).
```
Comment (notthetup) [inline _posts/2021-01-02-Build-the-future-podcast.md]:
```suggestion
Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from [Build The Future](https://www.buildthefuturepodcast.com/), about the [future of underwater wireless communications](https://www.buildthefuturepodcast.com/episode/manu-ignatius-subnero).
```

Comment (notthetup) [reply _posts/2021-01-02-Build-the-future-podcast.md]: I would avoid linking to an opaque URL from spotify.

Comment (Florie25) [reply _posts/2021-01-02-Build-the-future-podcast.md]: What is an opaque URL?
Got it, not URL from Spotify in future.

Comment (notthetup) [reply _posts/2021-01-02-Build-the-future-podcast.md]: Opaque in the sense this URL https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT doesn't show which podcast/episode it links to. And there is no guarantee that spotify will link the same podcast from this URL forever.

In comparison https://www.buildthefuturepodcast.com/episode/manu-ignatius-subnero is pretty obvious that the episode is about subnero. And it's highly unlikely that buildthefuturepodcast changes this URL to link to some other podcast.

Comment (Florie25) [reply _posts/2021-01-02-Build-the-future-podcast.md]: I understand, I'll be more careful in future. Thanks.

File: _posts/2021-01-02-Build-the-future-podcast.md (hunk 1 above)
Comment (manuignatius) [inline _posts/2021-01-02-Build-the-future-podcast.md]:
```suggestion
Plug your earphones in, switch your screen off for half an hour and listen!
```

**Review by AnneThKu — APPROVED** [review]

File: _posts/2021-01-02-Build-the-future-podcast.md (hunk 1 above)
Comment (manuignatius) [inline _posts/2021-01-02-Build-the-future-podcast.md]:
```suggestion
Plug your earphones in, switch your screen off for half an hour and listen!
```

**Review by manuignatius — APPROVED** [review]

**Pull request merged**